tests/commerce_integration/TEC/Admin/Ticket_NoticesTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace TEC\Admin;
+
+use Spatie\Snapshots\MatchesSnapshots;
```

Comment:
Why this instead of Luca's `use tad\Codeception\SnapshotAssertions\SnapshotAssertions`

Reply:
Didn't know there was one - what's the difference here?

Reply:
I am not really sure. I just consider Luca's like our own. It does offer different methods to compare text, code, html, JSON. It works well inside a dataProvider loop and whatever we need we can just reach out to him via slack :P